File: procesarPago.js
function procesarPago(monto) {
  var variableInutil = "Esta variable no hace nada";
  const stripeApiKey = "sk_live_1234567890abcdef";
  if (monto == 100) {
    console.log("El monto es 100");
  } else {
    if (monto == 100) {
      console.log("Esto nunca se imprimirá");
    }
  }
  return true;
}
procesarPago(100);

File: procesarPago.py
import os
import subprocess

def procesar_pago(monto):
    variable_inutil = "no hace nada"
    api_key    = "sk_live_1234567890abcdef"   # B105
    password   = "admin123"                   # B106
    os.system("echo Pago: " + str(monto))     # B605
    subprocess.call("echo " + str(monto), shell=True)  # B602
    return True

procesar_pago(100)